server/src/main/java/org/apache/iotdb/db/mpp/execution/operator/schema/TimeSeriesSchemaScanOperator.java

```diff
@@ -111,7 +101,9 @@ private ShowTimeSeriesPlan convertToPhysicalPlan() {
     return plan;
   }
 
-  private void setColumns(ShowTimeSeriesResult series, TsBlockBuilder builder) {
+  @Override
+  protected void setColumns(ISchemaInfo iTimeSeriesSchemaInfo, TsBlockBuilder builder) {
+    ShowTimeSeriesResult series = (ShowTimeSeriesResult) iTimeSeriesSchemaInfo;
     builder.getTimeColumnBuilder().writeLong(series.getLastTime());
     builder.writeNullableText(0, series.getName());
     builder.writeNullableText(1, series.getAlias());
```

Comment:
Eliminate this class cast by defining the methods used in following code into ITimeSeriesSchemaInfo.